config.c

```diff
@@ -1266,6 +1267,20 @@ static int readConfigFile(const char *configFile, struct logInfo *defConfig)
 					message(MESS_DEBUG, "compress_prog is now %s\n",
 						newlog->compress_prog);
 
+			 		compresscmd_base=strdup(basename(newlog->compress_prog));
+			 		fprintf(stdout, "compress_ext is %s\n", newlog->compress_prog);
+			 		i=0; /* have to check whether we may do this! */
```

Comment:
Why assigning zero to the variable ```i``` would be a problem?  Is the comment misplaced?